product_docs/docs/lasso/4/describe.mdx

```diff
@@ -4,2237 +4,2092 @@ navTitle: Gathered data details
+**Report output:**
 
-**Depth:** surface
+  * File `/linux/limits.data`: Content of the `limits.conf` file
 
+**Depth:** Surface
 
-**Security Impact:** *low* &mdash; 
-no known security impact.
+**Security impact:** Low &mdash; 
+No known security impact.
 
 ### Processor usage statistics (`linux_mpstat`)
 
 Processor statistics from the `mpstat` command.
 
-Report output:
+**Report output:**
 
-  * file `/linux/mpstat.data`: Output from 'mpstat -P ALL 1 10'
+  * File `/linux/mpstat.data`: Output from `mpstat -P ALL 1 10`
 
-**Depth:** surface
+**Depth:** Surface
 
-
-**Security Impact:** *low* &mdash; 
-no known security impact.
+**Security impact:** Low &mdash; 
+No known security impact.
 
 ### Network interfaces (`linux_network_interfaces`)
 
 Network interface information from the `ip` and `ifconfig` commands.
 
-Report output:
-
-  * file `/linux/ifconfig.data`: Output from 'ifconfig'
-  * file `/linux/ip_address_list.data`: Output from 'ip address list'
+**Report output:**
 
-**Depth:** surface
+  * File `/linux/ifconfig.data`: Output from `ifconfig`
+  * File `/linux/ip_address_list.data`: Output from `ip address list`
 
+**Depth:** Surface
 
-**Security Impact:** *low* &mdash; 
-no known security impact.
+**Security impact:** Low &mdash; 
+No known security impact.
 
 ### Installed packages via rpm or dpkg (`linux_packages_info`)
 
-Information about the system packages installed using `rpm` or `dpkg`
-
-Report output:
+Information about the system packages installed using `rpm` or `dpkg`.
 
-  * file `/linux/packages-dpkg.data`: List of packages installed using `dpkg`
-  * file `/linux/packages-rpm.data`: List of packages installed using `rpm`
+**Report output:**
 
-**Depth:** surface
+  * File `/linux/packages-dpkg.data`: List of packages installed using `dpkg`
+  * File `/linux/packages-rpm.data`: List of packages installed using `rpm`
 
+**Depth:** Surface
 
-**Security Impact:** *low* &mdash; 
-no known security impact.
+**Security impact:** Low &mdash; 
+No known security impact.
 
 ### PostgreSQL disk layout (`linux_postgresql_disk_layout`)
 
 List all files in the PostgreSQL data directory using `find` for
-links and `ls` for files
+links and `ls` for files.
 
-Report output:
+**Report output:**
 
-  * file `/linux/pg_ls.data`: List of files inside the data directory
-  * file `/linux/pg_links.data`: List of links inside the data directory
+  * File `/linux/pg_ls.data`: List of files inside the data directory
+  * File `/linux/pg_links.data`: List of links inside the data directory
 
-**Depth:** surface
+**Depth:** Surface
 
-
-**Security Impact:** *low* &mdash; 
-no known security impact.
+**Security impact:** Low &mdash; 
+No known security impact.
 
 ### SELinux (`linux_sestatus`)
 
-SELinux status from `sestatus`
-
-Report output:
+SELinux status from `sestatus`.
 
-  * file `/linux/sestatus.data`: Output from 'setstatus'
+**Report output:**
 
-**Depth:** surface
+  * File `/linux/sestatus.data`: Output from `setstatus`
```

Comment:
This is a typo in the original file (and the command's own documentation) - the actual command is **se**status, not se**t**status; it is correct above.

Reply:
I fixed this in the description (output from `sestatus`